sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -152,10 +174,19 @@ export class ServiceBusClient {
   }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in peekLock mode.
+   *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
```

Comment:
In case of sessions, the lock is no longer on the message, but on the entire session.

Reply:
Modified the top bit to mention session lock, but kept referring to lock throughout.